PlexPy.py

```diff
@@ -49,6 +49,12 @@ def main():
 
     plexpy.PROG_DIR = os.path.dirname(plexpy.FULL_PATH)
     plexpy.ARGS = sys.argv[1:]
+    plexpy.BACKUP_DIR = os.path.join(plexpy.PROG_DIR, 'backups')
+
+    try:
```

Comment:
This should be moved to the same place in the code where the other directories are created.
https://github.com/Hellowlol/plexpy/blob/master/plexpy/__init__.py#L128